### ISSUE TITLE
ValueError: invalid literal for int() with base 10: '0.0'

### ISSUE THREAD

**Issue by pcoccoli** [IBM/firepit#104]
From this line:
```
File \"/usr/local/lib/python3.9/site-packages/firepit/aio/ingest.py\", line 308, in translate
    df[txf_col] = df[txf_col].dropna().astype('int')
```
Can't convert the *string* "0.0" to int. E.g. in ipython:
```
In [26]: int(0.0)
Out[26]: 0

In [27]: int("0.0")
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Input In [27], in <module>
----> 1 int("0.0")

ValueError: invalid literal for int() with base 10: '0.0'

```

**Comment by pcoccoli** [comment]
Should be fixed in commit 7ab26a9

**Comment by pcoccoli** [comment]
Fixed in 2.3.26
